<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>339</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>493</width>
    <height>339</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>493</width>
    <height>339</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit Property</string>
  </property>
  <widget class="QLabel" name="editPropertyLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Text</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="editField">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>471</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>166</width>
     <height>27</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
